examples/blog/api.ttl

```diff
@@ -0,0 +1,79 @@
+@base <http://localhost:9000/api/schema/>.
+
+@prefix hydra: <http://www.w3.org/ns/hydra/core#>.
+@prefix code: <https://code.described.at/>.
+@prefix hydrabox: <http://hydra-box.org/schema/>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xml: <http://www.w3.org/XML/1998/namespace>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+
+<Blog> a hydra:Class;
+  hydra:search <blog#search>;
+  hydra:supportedOperation
+    <blog#get>,
+    <blog#post>.
+
+<blog#get> a hydra:SupportedOperation;
+  hydrabox:variables <blog#search>;
+  hydra:expects <TimeRange>;
+  hydra:method "GET";
+  hydra:returns <Result>;
```

Comment:
This `hydra:returns` also strikes me as odd. A get method would likely return what it annotates. `Blog` in this case